exercises/all-your-base/example.rb

```diff
@@ -1,17 +1,22 @@
 module BookKeeping
-  VERSION = 1
+  VERSION = 2
 end
 
 class BaseConverter
   def self.convert(base_from, number_array, base_to)
-    return if number_array.any?{|number| number < 0 || number >= base_from}
-    return if base_from <= 1 || base_to <= 1
+    fail ArgumentError if invalid_inputs?(base_from, number_array, base_to)
```

Comment:
I tried to change as little as possible here to get tests to pass. Does this look okay?